swaglyrics_backend/issue_maker.py

```diff
@@ -234,12 +239,18 @@ def check_song(song, artist):
     except KeyError:
         return False
     if data:
-        print(data[0]['artists'][0]['name'], data[0]['name'])
-        if data[0]['name'] == song and data[0]['artists'][0]['name'] == artist:
+        track = data[0]
+        print(track['artists'][0]['name'], track['name'])
+        if track['name'] == song and track['artists'][0]['name'] == artist:
             print(f'{song} and {artist} legit on Spotify')
+            metadata = requests.get(f'https://api.spotify.com/v1/audio-features/{track["id"]}', headers=headers)
+            if metadata["instrumentalness"] > 0.45 and metadata["speechiness"] < 0.04:
```

Comment:
I thought the bar for instrumentalness was 0.5 according to the API docs?

Reply:
I found that there was some overlap. In my experiment this was the optimum value. Each of the attributes has a different scale which is why the threshold for speechiness is so low.

Reply:
Okay, no problem then